docs/contributing/documentation-guidelines/index.md

```diff
@@ -13,9 +15,46 @@ Examples of larger changes include:
 - Adding new pages with a large amount of detail, such as a tutorial
 - Re-organization of the existing documentation structure
 
-In terms of style, you should refer to the [Google developer documentation style guide](https://developers.google.com/style) as much as possible. Reading the [Highlights page](https://developers.google.com/style/highlights) of that guide is recommended, but if not then the key points below should be noted.
+## Style guide
+
+You should refer to the [Google developer documentation style guide](https://developers.google.com/style) as much as possible. Reading the [Highlights page](https://developers.google.com/style/highlights) of that guide is recommended, but if not then the key points below should be noted.
 
 - [Use standard American English spelling](https://developers.google.com/style/spelling) and punctuation.
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## Tips
+
+### How to preview your modification
+
+There are two ways to preview your modification on a documentation website.
+
+#### 1. Using GitHub Actions workflow
+
+Follow the steps below.
+
+1. Create a pull request to the repository.
+2. Add the `documentation` label from the sidebar (See below figure).
+3. Wait for a couple of minutes, and the `github-actions` bot will notify the URL for the pull request's preview.
+
+![documenatation_label](images/documentation_label_for_pull_request.png){ width="800" }
```

Comment:
[nits] Could you use dash-case for files?

```suggestion
![documenatation_label](images/documentation-label-for-pull_request.png){ width="800" }
```

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/e5d9709efe8d42eb56239c31ac4b6864e521c900
Done!